src/massive/munit/Assert.hx

```diff
@@ -226,4 +226,22 @@ class Assert
 	{
 		throw new AssertionException(msg, info);
 	}
+    
+    /**
+     * Assert that an expectation was thrown. Can expect strings and non-strings.
+     *
+     * @param   code                a function which should throw an exception 
+     * @return                      the exception that was thrown      
+     * @throws  AssertionException  if no expectation is thrown
+     */
+    public static function throws(code:Dynamic):Dynamic
```

Comment:
- I'll take a look. I didn't know about `posinfo`.
- You're not specifying an expected exception any more. `throws` returns the caught exception. The common use-case is to call something like `var caught = Assert.throws(...)` and then make further assertions against `caught`. This also keeps the framework a little leaner.
- It looks like you're using tabs (I thought everyone uses spaces because of cross-IDE consistency). I'll tabify my indentation.